Kernel/Arch/aarch64/SafeMem.cpp

```diff
@@ -9,23 +9,34 @@
 
 namespace Kernel {
 
-bool safe_memset(void* dest_ptr, int c, size_t n, void*&)
+bool safe_memset(void* dest_ptr, int c, size_t n, void*& fault_at)
 {
     // FIXME: Actually implement a safe memset.
-    memset(dest_ptr, c, n);
+    auto* dest = reinterpret_cast<u8*>(dest_ptr);
```

Comment:
`bit_cast` for these three

Reply:
A simple `static_cast` was enough in this case.